docs/content/en/docs/operator-manual/piped/installation.md

```diff
@@ -93,7 +107,23 @@ Most part is identical to the previous way, but some part is slightly different.
     --set rbac.scope=namespace
   ```
 
-### Installing on single machine
+#### In case on OpenShift less than 4.2
+Containers on OpenShift run using arbitrary Linux users. What we don't know the UID in advance causes a couple of problems.
+To prevent you from such problems, you should set additional options to the previous `helm upgrade` command.
+
```

Comment:
nit: I think we should not talk about "couple of problems", this causes anxiety and ambiguity.

How about this: 

OpenShift uses an arbitrarily assigned user ID when it starts a container. Starting from OpenShift 4.2, it also inserts that user into `/etc/passwd` for using by the application inside the container, but before that version, the assigned user is missing in that file. That blocks working of `gcr.io/pipecd/piped` image.
Therefore if you are running on OpenShift with a version before 4.2, please use `gcr.io/pipecd/piped-okd` image with the following command:

Reply:
Tell you what, you're quite right. I was guessing users don't have to know how the underlying works, but you know what? They could get anxiety as you were saying. Let me employ what you propose as it is 👍